src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -30,6 +30,8 @@ static DHCPLease *dhcp_lease_free(DHCPLease *lease) {
         return mfree(lease);
 }
 
+_SD_DEFINE_POINTER_CLEANUP_FUNC(DHCPLease, dhcp_lease_free);
```

Comment:
this should be used in public headers only. internally please use DEFINE_TRIVIAL_CLEANUP_FUNC() instead